core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -288,7 +297,6 @@ enum class UniqueType(
     RequiresBuildingInAllCities("Requires a [buildingFilter] in all cities", UniqueTarget.Building),
     @Deprecated("as of 4.10.17", ReplaceWith("Only available <if [buildingFilter] is constructed in at least [positiveAmount] of [All] cities>"))
     RequiresBuildingInSomeCities("Requires a [buildingFilter] in at least [positiveAmount] cities", UniqueTarget.Building),
-    CanOnlyBeBuiltInCertainCities("Can only be built [cityFilter]", UniqueTarget.Building),
```

Comment:
Please deprecate and add new unique, not remove/replace. Side tangent: I know the unique is similar OnlyAvailableWhen, but it probably should still be down here for consistency when reading through the list of uniques

Reply:
Put back in old Unique with Depreciated notification. Can add in version once more stable